src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt

```diff
@@ -0,0 +1,168 @@
+
+    override fun latestUpdatesFromElement(element: Element): SAnime = getAnimeFromAnimeElement(element)
+
+    private fun getAnimeFromAnimeElement(element: Element): SAnime {
+        val anime = SAnime.create()
+        anime.setUrlWithoutDomain(element.select("div > a").first().attr("href"))
+        anime.thumbnail_url = element.select("div > a > div.limit > img").first().attr("src")
+        anime.title = element.select("div > a > div.tt > h2").text()
+        return anime
+    }
+    override fun latestUpdatesNextPageSelector(): String = "div.hpage > a.r"
+
+    override fun latestUpdatesRequest(page: Int): Request = GET("$baseUrl/anime/?page=$page&status=ongoing&sub=&order=update")
+
+    override fun latestUpdatesSelector(): String = "div.listupd > article"
+
+    override fun popularAnimeFromElement(element: Element): SAnime = getAnimeFromAnimeElement(element)
+
+    override fun popularAnimeNextPageSelector(): String = "div.hpage > a.r"
+
+    override fun popularAnimeRequest(page: Int): Request = GET("$baseUrl/anime/?page=$page")
+
+    override fun popularAnimeSelector(): String = "div.listupd > article"
+
+    override fun searchAnimeFromElement(element: Element): SAnime = getAnimeFromAnimeElement(element)
+
+    override fun searchAnimeNextPageSelector(): String = "a.next.page-numbers"
+
+    override fun searchAnimeRequest(page: Int, query: String, filters: AnimeFilterList): Request {
+        // filter and stuff in v2
+        return GET("$baseUrl/page/$page/?s=$query")
+    }
+
+    override fun searchAnimeSelector(): String = "div.listupd > article"
+
+    override fun videoFromElement(element: Element): Video {
+        val res = client.newCall(GET(element.attr("href"))).execute().asJsoup()
+        val scr = res.select("script:containsData(dlbutton)").html()
+        var url = element.attr("href").substringBefore("/v/")
+        val firstString = scr.substringAfter(" = \"").substringBefore("\" + ")
+        val num1 = scr.substringAfter("+ (").substringBefore(" % ").toInt()
+        val num2 = scr.substringAfter(" % ").substringBefore(" + ").toInt()
+        val num4 = scr.substringAfter(" % ").substringBefore(") + ").substringAfter(" % ").toInt()
+        val lastString = scr.substringAfter(") + \"").substringBefore("\";")
+        val num = (num1 % num2) + (num1 % num4)
+        url += firstString + num.toString() + lastString
+        val quality = with(url) {
+            when {
+                contains("1080p") -> "1080p"
+                contains("720p") -> "720p"
+                contains("480p") -> "480p"
+                contains("360p") -> "360p"
+                else -> "Unknown Resolution"
+            }
+        }
+        return Video(url, quality, url, null)
+    }
+
+    override fun videoListSelector(): String = "div.mctnx > div > div > a:nth-child(3)"
+
+    override fun videoUrlParse(document: Document) = throw Exception("not used")
+
```

Comment:
you need to override the function `List<Video>.sort()` in order for your quality preference to work.
example: https://github.com/jmir1/aniyomi-extensions/blob/9f791a0684940c40d8c8a6ddaaec5db29a0aac1e/src/en/gogoanime/src/eu/kanade/tachiyomi/animeextension/en/gogoanime/GogoAnime.kt#L175-L191

Reply:
Uh thats why my resolution didnt sorted correctly